app/views/layouts/application.html.erb

```diff
@@ -10,6 +10,8 @@
 
     <%= render "donations/new" %>
 
+    <%= render "events/new" %>
```

Comment:
Unless we need to create events from every page, we should move this to a more specific template. 😀